book/intro.qmd

```diff
@@ -7,57 +7,57 @@
 |                | of land use in urban environment                 |
 +----------------+--------------------------------------------------+
 | **Output**     | Prediction of the quality of life indicators     |
-|                | following a modelled scenarios of development    |
+|                | following modelled scenarios of development      |
 |                |                                                  |
-|                | study case: Tyne and Wear County (Local          |
-|                | Authorities: Gateshead, Newcastle Upon Tyne,     |
-|                | North Tyneside, South Tyneside, Sunderland)      |
+|                | Our case study is Tyne and Wear county,          |
+|                | comprising the following local authorities:      |
+|                | Gateshead, Newcastle upon Tyne, North Tyneside,  |
+|                | South Tyneside, and Sunderland.                  |
 +----------------+--------------------------------------------------+
-| **How**        | Creating key **indicators** for assessing a      |
+| **How**        | Creating key indicators for assessing a          |
 |                | *baseline* scenario and *future* scenarios       |
 +----------------+--------------------------------------------------+
 | **Duration**   | 6 months                                         |
 +----------------+--------------------------------------------------+
 
 ## Project aims
 
-The project aims to provide insight into the impact of policies affecting land use in cities across the UK, piloting on the case of Tyne and Wear. It:
+The project aims to provide insight into the impact of land use policies in cities across the UK, piloting on the case of Tyne and Wear. It:
 
-1.	Derives indicators of quality of life.
-2.	Develops machine learning models able to predict the impact of land use changes on such indicators.
-3.	And inversely, develops a neural network able to predict the required land use change to reach target levels of QoL indicators.
+1. Derives indicators of quality of life.
+2. Develops machine learning models which can predict the impact of land use changes on such indicators.
+3. Inversely, develops a neural network able to predict the required land use change to reach target levels of quality of life indicators.
 
 All these technological components are presented in an interactive tool allowing quick and easy exploration of impacts aimed at policymakers.
 
```

Comment:
In every subsection of the Intro page, there is one line on the webapp.

Does it make sense to factorise this out into its own subsection at the bottom of the page?

Reply:
This mostly mirrors whatever was sent to Kelin to be published on Turing site which didn't allow us a custom section. So yeah, if you think it makes more sense to have a dedicated section, go for it.